aes/ta/include/aes_ta.h

```diff
@@ -79,4 +81,6 @@
  */
 #define TA_AES_CMD_CIPHER		3
 
+#define CMD_AUTH_AES_INIT		4
```

Comment:
I think `TA_AES_CMD_AUTHENC_INIT` is more consistent.
Please add a description of the TA function with the above as examples.